bundles/org.eclipse.passage.lbc.api/src/org/eclipse/passage/lbc/internal/api/persistence/PersistableLicense.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lbc.internal.api.persistence;
+
+import java.io.IOException;
+import java.util.function.Supplier;
+
+/**
+ * @since 1.0
+ */
+public abstract class PersistableLicense implements Supplier<BoundLicense> {
+
+	private final BoundLicense license;
+
+	public PersistableLicense(BoundLicense license) {
+		this.license = license;
```

Comment:
There is a life-saving practice
```
Objects.requireNotNull(license, "PersistableLicense::license"");  //$NON-NLS-1$
```

It's a developer-aimed error, thus is can avoid i18n